### ISSUE TITLE
[ERROR] run GATKSVPipelineBatch.wdl with /gatk-sv/inputs/GATKSVPipelineBatch.ref_panel_1kg.json

### ISSUE THREAD

**Issue by NewOrange** [broadinstitute/gatk-sv#32]
### Description 
***
I try to tun GATKSVPipelineBatch.wdl with /gatk-sv/inputs/GATKSVPipelineBatch.ref_panel_1kg.json，and get such a error:
```
java.lang.RuntimeException: Failed to evaluate 'BAF_file' (reason 1 of 1): Evaluating select_first([baf_out]) failed: select_first was called with 1 empty values. We needed at least one to be filled.
        at cromwell.engine.workflow.lifecycle.execution.keys.ExpressionKey.processRunnable(ExpressionKey.scala:29)
        at cromwell.engine.workflow.lifecycle.execution.WorkflowExecutionActor.$anonfun$startRunnableNodes$7(WorkflowExecutionActor.scala:536)
        at cats.instances.ListInstances$$anon$1.$anonfun$traverse$2(list.scala:74)
        at cats.instances.ListInstances$$anon$1.loop$2(list.scala:64)
        at cats.instances.ListInstances$$anon$1.$anonfun$foldRight$1(list.scala:64)
        at cats.Eval$.loop$1(Eval.scala:338)
        at cats.Eval$.cats$Eval$$evaluate(Eval.scala:368)
        at cats.Eval$Defer.value(Eval.scala:257)
        at cats.instances.ListInstances$$anon$1.traverse(list.scala:73)
        at cats.instances.ListInstances$$anon$1.traverse(list.scala:12)
        at cats.Traverse$Ops.traverse(Traverse.scala:19)
        at cats.Traverse$Ops.traverse$(Traverse.scala:19)
        at cats.Traverse$ToTraverseOps$$anon$2.traverse(Traverse.scala:19)
```
### Related documents
***
+ https://github.com/broadinstitute/gatk-sv/blob/master/inputs/GATKSVPipelineBatch.ref_panel_1kg.json
+ https://github.com/broadinstitute/gatk-sv/blob/master/wdl/Module00c.wdl
+ https://github.com/broadinstitute/gatk-sv/blob/master/wdl/GATKSVPipelinePhase1.wdl
### How to fix it? 
***
I observe  ` "GATKSVPipelineBatch.GATKSVPipelinePhase1.run_matrix_qc": "true",` in GATKSVPipelineBatch.ref_panel_1kg.json without any statement about baf_files or gvcfs or snp_vcfs.
I want to know if I should give baf_files or gvcfs or snp_vcfs to solve this problem.
***
I am anxiously waiting for your reply. Thank you very much.

**Comment by mwalker174** [comment]
Hi @NewOrange, the Batch pipeline is currently not working for BAF. We are working on an update for GATK that will resolve an issue with the 1KGP gVCFs.

**Comment by NewOrange** [comment]
Hi @mwalker174 , thank you so much for your apply. I am looking forward to your update.

**Comment by mwalker174** [comment]
This was addressed in #51
